python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -392,6 +433,170 @@ class NearestNeighbors(Base):
         )
 
         self.handle.sync()
+        
+        if not _output_cumlarray:
+            I_ndarr = I_ndarr.to_output(out_type)
+            if return_distance:
+                D_ndarr = D_ndarr.to_output(out_type)
+
+        # drop first column if using training data as X
+        # this will need to be moved to the C++ layer (cuml issue #2562)
+        if use_training_data:
+            if out_type == 'cupy' or out_type == 'numpy' \
```

Comment:
Nitpick: `if out_type in { 'cupy', 'numpy', 'numba', 'cumlarray' }` is a slightly cleaner way to do this